### PR TITLE
flash bump

### PR DESCRIPTION
**Pull request by mictadlo**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/flash/meta.yaml [hunk 1]
```diff
@@ -21,13 +24,10 @@ requirements:
...
+    - flash2 -v
 
 about:
-  home: https://ccb.jhu.edu/software/FLASH/
+  home: https://github.com/dstreett/FLASH2
   license: GPLv3+
-  summary: 'Merge mates from fragments that are shorter than twice the read length'
+  summary: 'Merge paired-end reads from fragments that are shorter than twice the read length'
 
-extra:
```
Comment (bgruening) [inline recipes/flash/meta.yaml]: Why have you removed this? its still the same tool isn't it?
Is there a publication which we can reference?

File: recipes/flash/meta.yaml [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
+{% set version = "2.2.00" %}
 
 package:
-  name: flash
-  version: '1.2.11'
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/flash/meta.yaml]: start with 0

**Comment by marcelm** [comment]
I’m uncertain as to the status of how FLASH2 relates to FLASH. Looking at the [GitHub repository](https://github.com/dstreett/FLASH2/), it seems that the project is an independent one (not from the original authors).

Perhaps @dstreett could explain?

According to dstreet/FLASH2#2, the behavior is also slightly different, so perhaps it woud be an idea to put this into a separate recipe named `flash2`.

**Comment by bgruening** [comment]
It it is really a separate "tool" than a new recipe would be preferred, I agree.

**Comment by mictadlo** [comment]
I created a new PR https://github.com/bioconda/bioconda-recipes/pull/9103

**Pull request closed without merging**